packages/jmix-addon-data-tools/src/entity-inspector/entityRestore.ts

```diff
@@ -1,23 +1,37 @@
 import { useMutation, gql } from "@apollo/client";
-import { useCallback, useMemo } from "react";
+import { notifications, NotificationType } from "@haulmont/jmix-react-antd";
+import { useCallback } from "react";
+import { useIntl } from "react-intl";
 
 type RestoreEntitiesVariables = {
   className: string;
   ids: string[];
 }
 
-export function useEntityRestore(restoreEntitesVariables: RestoreEntitiesVariables) {
+type RestoreData = {
+  restoreEntities: number
```

Comment:
Looks like field should be named `restoredEntities`

Reply:
@vadimbasko  It's API's naming